spec/locales-currencies-tz.html

```diff
@@ -213,6 +222,29 @@ <h1>DefaultTimeZone ( )</h1>
         The DefaultTimeZone abstract operation returns a String value representing the valid (<emu-xref href="#sec-isvalidtimezonename"></emu-xref>) and canonicalized (<emu-xref href="#sec-canonicalizetimezonename"></emu-xref>) time zone name for the host environment's current time zone.
       </p>
     </emu-clause>
+
+    <emu-clause id="sec-availablecanonicaltimezones" type="implementation-defined abstract operation">
+      <h1>
+        AvailableCanonicalTimeZones (
```

Comment:
```suggestion
        AvailableCanonicalTimeZoneIdentifiers (
```

With Temporal approaching Stage 4, "TimeZone" will soon be ambiguous: does it mean string identifier or a `Temporal.TimeZone` object?

Therefore @gibson042 and I are planning to rename all identifier-related AOs with an "Identifier" suffix. Given that this is a new AO, should we rename it here too to save churn later?

Reply:
how about do those after merge this PR. Temporal is not going to be ES2023 but this will be

Reply:
OK, that's fine with me. Not a big deal either way.